loopy/translation_unit.py

```diff
@@ -54,24 +54,7 @@
 """
 
 
-def find_in_knl_callable_from_identifier(
```

Comment:
@kaushikcfd Any reason this should be kept?

Reply:
Thanks, this was dead code, that correpsonded to logic in a previous iteration of Program.